README.md

```diff
@@ -37,6 +37,36 @@ The `azure.blob` plugin stores data in Azure Blob Storage.
 1. [Copy the connection string][connstring] for the storage account. Then set it as an environment variable named 
     `AZURE_STORAGE_CONNECTION_STRING`.
 
+## Secrets
+
+Secrets plugins allow Porter to inject secrets into credential sets.
+
+For example, your team may have a shared key vault with a database password, you can use the keyvault plugin
```

Comment:
nit: sentence reads slightly awkward.  Perhaps we can add an `if` here, for: `For example, if your team has a shared...`?